packages/web-runtime/src/pages/resolvePublicLink.vue

```diff
@@ -82,7 +82,7 @@ import {
   PublicSpaceResource
 } from 'web-client/src/helpers'
 import isEmpty from 'lodash-es/isEmpty'
-import { useLoadTokenInfo } from '../composables/tokenInfo'
+import { useLoadTokenInfo } from 'web-runtime/src/composables/tokenInfo'
```

Comment:
Maybe comment here, that the import path is important for the cern import alias...